src/dataframe/dataframe.jl

```diff
@@ -16,7 +16,7 @@ DataFrame(kwargs..., copycols::Bool=true)
 DataFrame(columns::AbstractVecOrMat, names::Union{AbstractVector, Symbol};
           makeunique::Bool=false, copycols::Bool=true)
 
-DataFrame(table; copycols::Bool=true)
+DataFrame(table; copycols=nothing)
```

Comment:
I think it is relevant to show that this kwarg can be `Bool` here:
```suggestion
DataFrame(table; copycols::Union{Bool, Nothing}=nothing)
```